test/functional/apps/management/ccs_compatibility/_data_views_ccs.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const kibanaServer = getService('kibanaServer');
+  const testSubjects = getService('testSubjects');
+  const es = getService('es');
+  const PageObjects = getPageObjects(['settings', 'common', 'header']);
+  const security = getService('security');
+
+  describe('data views ccs', function () {
+    before(async function () {
+      await security.testUser.setRoles(['kibana_admin']);
+    });
+
+    describe('index pattern wizard ccs', () => {
+      describe('remote cluster only', async () => {
+        beforeEach(async function () {
+          // delete .kibana index and then wait for Kibana to re-create it
```

Comment:
Just a nit but this comment appears to be an old leftover from calling es_archiver/empty_kibana. (in a couple of places)

Reply:
I see what you mean. Actually we are calling `kibanaServer.savedObjects.cleanStandardList()` in the `afterEach` hook to reset the environment to having no index patterns. And then we navigate back to the page.